base/gmp.jl

```diff
@@ -978,7 +1014,7 @@ for (fJ, fC) in ((:+, :add), (:-, :sub), (:*, :mul), (://, :div))
     fC! = Symbol(fC, :!)
     @eval begin
         ($fC!)(x::Rational{BigInt}, y::Rational{BigInt}) = $fC!(x, x, y)
-        ($fJ)(x::Rational{BigInt}, y::Rational{BigInt}) = $fC!(unsafe_rational(BigInt(), BigInt()), x, y)
+        (Base.$fJ)(x::Rational{BigInt}, y::Rational{BigInt}) = $fC!(unsafe_rational(BigInt(), BigInt()), x, y)
```

Comment:
It's unlclear to me if `Base.` is needed here.. can anyone help? 😄

Reply:
I think it is indeed needed because you did not import  `+`, `-`, `*` and `/` in the `MPQ` module in which those methods are defined. You can check what precise method is called with `@functionloc` or `@edit` if you want to see for yourself.

Reply:
Yes, the version with `Base` seems to be right